src/schema/objects/metadata.yaml

```diff
@@ -2112,6 +2239,47 @@ SampleOrigin:
   - bile
   - amniotic fluid
   - other biospecimen
+SamplePrimaryAntibody:
+  name: SamplePrimaryAntibody
+  description: |
+    Description(s) of the primary antibody used for immunostaining.
+    Either an [RRID](https://scicrunch.org/resources) or the name, supplier and catalogue
+    number of a commercial antibody.
+    For non-commercial antibodies either an [RRID](https://scicrunch.org/resources) or the
+    host-animal and immunogen used (for examples: `"RRID:AB_2122563"` or
+    `"Rabbit anti-Human HTR5A Polyclonal Antibody, Invitrogen, Catalog # PA1-2453"`).
+    MAY be an array of strings if different antibodies are used in each channel of the file.
```

Comment:
I'm not sure to understand your question.
The order of the array is only important in the sense that it describes the antibodies for channel of the file.
I did not provide an example here as it is the same as SampleStaining directly above with an example.

Reply:
I meant, does the order of the strings correspond to the order of the channels of the file? It sounds like that's the case,  so I was thinking that it would be good to be explicit about that. However, after looking at other fields that may be either a single value that applies to the whole file _or_ an array where each value applies to the corresponding slice/volume in the file, it doesn't seem like that's something we've required in the past. So I think I've talked myself out of requesting more information in this field's definition.